.env

```diff
@@ -0,0 +1,4 @@
+database: ""
```

Comment:
Any preferences on the database name? If not I can pick something simple like "authentication"

Reply:
we went with idam